handsontable/src/plugins/dropdownMenu/dropdownMenu.js

```diff
@@ -386,6 +387,24 @@ export class DropdownMenu extends BasePlugin {
     }
   }
 
+  /**
+   * Add custom shortcuts to the provided menu instance.
+   *
+   * @private
+   * @param {Menu} menuInstance The menu instance.
+   */
+  #addCustomShortcuts(menuInstance) {
+    const menuShortcutsCtrl = menuInstance.getKeyboardShortcutsCtrl();
+
+    menuShortcutsCtrl.addCustomShortcuts([{
+      keys: [['Control/Meta', 'A']],
+      callback: () => false,
+      group: SHORTCUTS_GROUP,
+      relativeToGroup: 'menu',
+      position: 'before',
+    }]);
+  }
```

Comment:
```suggestion
  #addCustomShortcuts(menuInstance) {
    menuInstance
      .getKeyboardShortcutsCtrl()
      .addCustomShortcuts([{
        keys: [['Control/Meta', 'A']],
        callback: () => false,
      }]);
  }
```
WDYT simplifying this?